test/models/event_log_test.rb

```diff
@@ -66,6 +66,14 @@ class EventLogTest < ActiveSupport::TestCase
     end
   end
 
+  test 'records role changes with the details of the roles' do
```

Comment:
I know the linter isn't complaining but double quotes are used everywhere else in this file. Can we change them here to keep it consistent?